modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/index/BasicIndexTest.java

```diff
@@ -761,6 +767,81 @@ public void testAllIndexesWithPersistence() throws Exception {
         }
     }
 
+    /** */
+    @Test
+    @WithSystemProperty(key = IGNITE_THROTTLE_INLINE_SIZE_CALCULATION, value = "1")
+    public void testInlineSizeChange() throws Exception {
```

Comment:
Also it would be great to reproduce in tests problems related to possible index corruption and other artifacts possible before that patch.

Reply:
ticket will be soon, but it still beyond the scope of this ticket.